DnsClient 1.8.0 causes reference conflicts in some .NET Standard 2.0 targets

DnsClient after 1.5.0+ imports a few packages that seem to target .NET 5, namely the following:

```
Successfully installed 'Microsoft.Win32.Registry 5.0.0' to UniSky
Successfully installed 'System.Security.AccessControl 5.0.0' to UniSky
Successfully installed 'System.Security.Principal.Windows 5.0.0' to UniSky
```

These packages cause version conflicts with the .NET Standard 2.0 library used by UWP. My app at least still appears to build and run, but I obviously can't say for sure if this is working correctly or not.

```
warning MSB3277: Found conflicts between different versions of "System.Security.Principal.Windows" that could not be resolved.
warning MSB3277: There was a conflict between "System.Security.Principal.Windows, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Security.Principal.Windows, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
warning MSB3277:     "System.Security.Principal.Windows, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Security.Principal.Windows, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
warning MSB3277:     References which depend on "System.Security.Principal.Windows, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.Principal.Windows.dll].
warning MSB3277:         C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.Principal.Windows.dll
warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.Principal.Windows.dll".
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.Principal.Windows.dll
warning MSB3277:     References which depend on or have been unified to "System.Security.Principal.Windows, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\PublicAssemblies\System.Security.Principal.Windows.dll].
warning MSB3277:         C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll".
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277:         C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll
warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll".
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Background\bin\x64\Debug\UniSky.Background.winmd
warning MSB3277:             Microsoft.UI.Xaml.Markup
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.ui.xaml\2.7.3\lib\uap10.0\Microsoft.UI.Xaml.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\win2d.uwp\1.28.2\lib\uap10.0\Microsoft.Graphics.Canvas.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\micaforuwp\0.2.13\lib\uap10.0.15138\MicaForUWP.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.animations\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Animations.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.media\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Media.dll
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\dnsclient\1.8.0\lib\netstandard2.0\DnsClient.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\fishyflip\4.1.0-alpha.12\lib\netstandard2.0\FishyFlip.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Services\bin\x64\Debug\UniSky.Services.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Models\bin\Debug\netstandard2.0\UniSky.Models.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Moderation\bin\Debug\netstandard2.0\UniSky.Moderation.dll
warning MSB3277:         C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll
warning MSB3277:           Project file item includes which caused reference "C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll".
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\dnsclient\1.8.0\lib\netstandard2.0\DnsClient.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\fishyflip\4.1.0-alpha.12\lib\netstandard2.0\FishyFlip.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Background\bin\x64\Debug\UniSky.Background.winmd
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Services\bin\x64\Debug\UniSky.Services.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Models\bin\Debug\netstandard2.0\UniSky.Models.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Moderation\bin\Debug\netstandard2.0\UniSky.Moderation.dll
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll
warning MSB3277:             Microsoft.UI.Xaml.Markup
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.ui.xaml\2.7.3\lib\uap10.0\Microsoft.UI.Xaml.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\win2d.uwp\1.28.2\lib\uap10.0\Microsoft.Graphics.Canvas.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\micaforuwp\0.2.13\lib\uap10.0.15138\MicaForUWP.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.animations\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Animations.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.media\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Media.dll
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277: Found conflicts between different versions of "System.Security.AccessControl" that could not be resolved.
warning MSB3277: There was a conflict between "System.Security.AccessControl, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" and "System.Security.AccessControl, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a".
warning MSB3277:     "System.Security.AccessControl, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was chosen because it was primary and "System.Security.AccessControl, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" was not.
warning MSB3277:     References which depend on "System.Security.AccessControl, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll].
warning MSB3277:         C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll".
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277:     References which depend on or have been unified to "System.Security.AccessControl, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" [].
warning MSB3277:         C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll
warning MSB3277:           Project file item includes which caused reference "C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll".
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Background\bin\x64\Debug\UniSky.Background.winmd
warning MSB3277:             Microsoft.UI.Xaml.Markup
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.ui.xaml\2.7.3\lib\uap10.0\Microsoft.UI.Xaml.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\win2d.uwp\1.28.2\lib\uap10.0\Microsoft.Graphics.Canvas.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\micaforuwp\0.2.13\lib\uap10.0.15138\MicaForUWP.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.animations\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Animations.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.media\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Media.dll
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\dnsclient\1.8.0\lib\netstandard2.0\DnsClient.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\fishyflip\4.1.0-alpha.12\lib\netstandard2.0\FishyFlip.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Services\bin\x64\Debug\UniSky.Services.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Models\bin\Debug\netstandard2.0\UniSky.Models.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Moderation\bin\Debug\netstandard2.0\UniSky.Moderation.dll
warning MSB3277:         C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll
warning MSB3277:           Project file item includes which caused reference "C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll".
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.win32.registry\5.0.0\ref\netstandard2.0\Microsoft.Win32.Registry.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\dnsclient\1.8.0\lib\netstandard2.0\DnsClient.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\fishyflip\4.1.0-alpha.12\lib\netstandard2.0\FishyFlip.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Background\bin\x64\Debug\UniSky.Background.winmd
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Services\bin\x64\Debug\UniSky.Services.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Models\bin\Debug\netstandard2.0\UniSky.Models.dll
warning MSB3277:             C:\Users\wamwo\source\repos\UniSky\UniSky.Moderation\bin\Debug\netstandard2.0\UniSky.Moderation.dll
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\mscorlib.dll
warning MSB3277:             Microsoft.UI.Xaml.Markup
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.ui.xaml\2.7.3\lib\uap10.0\Microsoft.UI.Xaml.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\win2d.uwp\1.28.2\lib\uap10.0\Microsoft.Graphics.Canvas.winmd
warning MSB3277:             C:\Users\wamwo\.nuget\packages\micaforuwp\0.2.13\lib\uap10.0.15138\MicaForUWP.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.animations\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Animations.dll
warning MSB3277:             C:\Users\wamwo\.nuget\packages\microsoft.toolkit.uwp.ui.media\6.1.1\lib\uap10.0.16299\Microsoft.Toolkit.Uwp.UI.Media.dll
warning MSB3277:             C:\Program Files (x86)\Microsoft SDKs\UWPNuGetPackages\microsoft.netcore.universalwindowsplatform\6.2.14\ref\uap10.0.15138\System.Security.AccessControl.dll
```

I don't really expect a fix for this one, especially given the issue is in an upstream package and it only really affects some pretty out there targets, but I thought I'd open an issue since it came up earlier

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -28,7 +28,7 @@
     <PackageVersion Include="MSTest" Version="3.10.4" />
     <PackageVersion Include="GitVersion.MSBuild" Version="6.4.0" />
     <PackageVersion Include="Duende.IdentityModel" Version="7.1.0" />
-    <PackageVersion Include="DnsClient" Version="1.8.0" />
+    <PackageVersion Include="DnsClient" Version="1.4.0" />
     <PackageVersion Include="Microsoft.Bcl.Memory" Version="9.0.9" />
     <PackageVersion Include="Microsoft.Bcl.HashCode" Version="6.0.0" />
     <PackageVersion Include="System.Buffers" Version="4.6.1" />

